draft-ietf-dprive-dnsoquic.md

```diff
@@ -983,21 +981,10 @@ Description:
 Reference:
 : {{!RFC7858}}{{?RFC8094}} This document
 
-The TEMPORARY assignment expires 13th December 2022.
-
 Additionally, IANA is requested to update the Description field for the
 corresponding TCP port 853 allocation to be 'DNS query-response protocol run
 over TLS' for consistency and clarity.
 
```

Comment:
Can we add some text to indicate clearly that this has already been reviewed by the port experts?
```suggestion
(UPDATE ON IANA REQUEST: THIS SENTENCE TO BE REMOVED BEFORE PUBLICATION) Review by the port experts on 13th December 2021 determined that the updates to the existing port allocation were acceptable and will be made when this document is approved for publication.
```